### ISSUE TITLE
doc build fails during debian packaging

### ISSUE THREAD

**Issue by gijzelaerr** [eduvpn/python-eduvpn-client#62]
```
# Sphinx version: 1.5.3
# Python version: 2.7.13 (CPython)
# Docutils version: 0.13.1 release
# Jinja2 version: 2.9.5
# Last messages:

# Loaded extensions:
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/sphinx/cmdline.py", line 295, in main
    opts.warningiserror, opts.tags, opts.verbosity, opts.jobs)
  File "/usr/lib/python2.7/dist-packages/sphinx/application.py", line 189, in __init__
    self.setup_extension(extension)
  File "/usr/lib/python2.7/dist-packages/sphinx/application.py", line 512, in setup_extension
    mod = __import__(extension, None, None, ['setup'])
  File "/usr/lib/python2.7/dist-packages/sphinx/builders/linkcheck.py", line 18, in <module>
    from requests.exceptions import HTTPError
  File "/usr/lib/python2.7/dist-packages/requests/__init__.py", line 53, in <module>
    from .packages.urllib3.contrib import pyopenssl
  File "/usr/lib/python2.7/dist-packages/requests/packages/__init__.py", line 52, in <module>
    vendored('urllib3')
  File "/usr/lib/python2.7/dist-packages/requests/packages/__init__.py", line 35, in vendored
    __import__(modulename, globals(), locals(), level=0)
  File "/usr/lib/python2.7/dist-packages/urllib3/__init__.py", line 8, in <module>
    from .connectionpool import (
  File "/usr/lib/python2.7/dist-packages/urllib3/connectionpool.py", line 35, in <module>
    from .connection import (
  File "/usr/lib/python2.7/dist-packages/urllib3/connection.py", line 71, in <module>
    class HTTPConnection(_HTTPConnection, object):
TypeError: Error when calling the metaclass bases
    cannot create 'NoneType' instances
```

**Comment by gijzelaerr** [comment]
build is working but files are not in doc package yet.

**Comment by gijzelaerr** [comment]
is included in 1.0rc8
